packages/api-kit/README.md

```diff
@@ -8,46 +8,27 @@ Software development kit that facilitates the interaction with the [Safe Transac
 
 ## Table of contents
 
+- [Documentation](#documentation)
 - [Installation](#installation)
-- [Build](#build)
-- [Tests](#tests)
-- [Initialization](#initialization)
-- [API Reference](#api-reference)
+- [Quick Start](#quick-start)
+- [Need Help or Have Questions?](#need-help-or-have-questions)
+- [Contributing](#contributing)
 - [License](#license)
-- [Contributors](#contributors)
 
-## <a name="installation">Installation</a>
+## Documentation
 
-Install the package with yarn or npm:
-
-```bash
-yarn install
-npm install
-```
-
-## <a name="build">Build</a>
-
-Build the package with yarn or npm:
-
-```bash
-yarn build
-npm run build
-```
+Head to the [API Kit docs](https://docs.safe.global/sdk/api-kit) to learn more about how to use this SDK.
 
-## <a name="tests">Tests</a>
+## Installation
 
-Create a `.env` file with environment variables. You can use the `.env.example` file as a reference.
-
-Test the package with yarn or npm:
+Install the package with yarn or npm:
 
 ```bash
-yarn test
-npm run test
+yarn add @safe-global/api-kit
+npm install @safe-global/api-kit
 ```
 
-## <a name="initialization">Initialization</a>
-
-### Initialize the SafeApiKit
+## Quick Start
```

Comment:
Below i will update the safeService const to apiKit

Reply:
Done